com.streetwriters.notesnook.yml

```diff
@@ -0,0 +1,75 @@
+app-id: com.streetwriters.notesnook
```

Comment:
`streetwriters.com` seems to be unrelated. Also the project list `https://notesnook.com/`, so I don't think the app-id meet the requirements.

Reply:
Notesnook runs under the parent company Streetwriters that owns the domain streetwriters.co

Can `co.streetwriters.notesnook` be used?

Reply:
`co` is not `com` mind you.

Reply:
But given the app data points to `    <url type="homepage">https://notesnook.com/</url>`

`com.notesnook.notesnook` seems to be the appropriate app-id in my books.

Reply:
Done.